src/index.html

```diff
@@ -11,7 +11,135 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <h1 class="title">HTML Form</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post" 
+      class="form">
+      <fieldset class="section">
+        <legend>Personal information</legend>
+        <label class="form_field">
+          Surname
+          <input 
+            type="text" 
+            name="Surname" 
+            required>
```

Comment:
```suggestion
          <input 
            type="text" 
            name="Surname" 
            required
          >
```
same - fix it everywhere